addons/calendar/models/calendar_attendee.py

```diff
@@ -104,6 +105,9 @@ def _send_mail_to_attendees(self, mail_template, force_send=False):
             :param mail_template: a mail.template record
             :param force_send: if set to True, the mail(s) will be sent immediately (instead of the next queue processing)
         """
+        for event, attendees in self.grouped('event_id'):
+            if not event._do_send_mail_status_update():
+                self -= self.env['calendar.attendee'].concat(*attendees)
```

Comment:
Isn't it? 
```suggestion
        for event, attendees in self.grouped('event_id').items():
            if not event._do_send_mail_status_update():
                self -= attendees
```

Reply:
Yep, was going to push a fix immediately after but I also noticed calendar actually removed their tests from __init__, probably on accident, at some point so now I have to make sure the tests work and reenable them ^^